flower/views/auth.py

```diff
@@ -268,19 +292,31 @@ def _OAUTH_AUTHORIZE_URL(self):
     def _OAUTH_ACCESS_TOKEN_URL(self):
         return f"{self.base_url}/v1/token"
 
+    @property
+    def _oauth_okta_scope(self):
+        return self.application.options.oauth2_okta_scope.split()
```

Comment:
OKTA supports adding different claims to the ID token based on the scopes set.
In our case we needed the "groups" claim and that is configured to require a "groups" scope in the request.